application-admintools-ui/src/main/resources/AdminTools/Code/Macros.xml

```diff
@@ -117,22 +117,22 @@
                     &lt;div class="dateFields">
                       #set ($defaultDateFormat = 'dd-MM-yyyy')
                       #set ($dateFormat = $xwiki.getXWikiPreference('dateformat', $defaultDateFormat))
-                      &lt;label for="${modalId}FromDate">$escapetool.xml($services.localization.render(
+                      &lt;label for="DownloadFilesModalFromDate">$escapetool.xml($services.localization.render(
                         'adminTools.dashboard.download.modal.date.from'))
                       &lt;/label>
                       #set ($dateFromParams = {
-                        'id': "${modalId}FromDate",
+                        'id': "DownloadFilesModalFromDate",
```

Comment:
Do you need the id here (and for toDate)? I think the date picker can be used without one

Reply:
Added the id for the labels.